src/au3wrap/au3wrapmodule.cpp

```diff
@@ -66,8 +66,7 @@ void Au3WrapModule::onInit(const muse::IApplication::RunMode&)
 
     Importer::Get().Initialize();
 
-    //! TODO AU4: we probably want to have preferences for temporary directory later
-    muse::String tempDir = globalConfiguration()->userAppDataPath().toString();
+    muse::String tempDir = projectConfiguration()->temporaryDir().toString();
```

Comment:
I believe that `temporaryDir` should be a global configuration method (i.e. setter and getter should be added). That is, the "temporary" directory is a property of the entire application, not the project.